airbyte-webapp/package.json

```diff
@@ -9,6 +9,8 @@
     "prepare": "cd .. && husky install airbyte-webapp/.husky",
     "prestart": "npm run generate-client",
     "start": "craco start",
+    "prestart:cloud": "npm run generate-client",
+    "start:cloud": "AB_ENV=frontend-dev node -r ./scripts/environment.js ./node_modules/.bin/craco start",
```

Comment:
:information_source: Using `AB_ENV` here (we use `AB` across infra more commonly as an abbreviation of Airbyte), to not collide with the standardized `ENV` variable for `production` vs `development`.